docs/learn-transactions.md

```diff
@@ -0,0 +1,62 @@
+---
+id: learn-transactions
+title: Transaction Types and Metering
+sidebar_label: Transaction Types and Metering
+---
+
+Depending on the pallets installed in a [Substrate](https://substrate.dev)-based chain like Polkadot or Kusama, different transactions are possible within the system. For example, one can propose referenda in the Governance pallet, verify identities in the Identities pallet, send tokens through the Balances pallet, shout remarks into the chain through the System pallet, and more. 
+
+Batching these transactions into blocks cannot work based on size like it does in a blockchain like Bitcoin's because smaller transactions might require more computation and vice versa - it's all about their computational effect. These transactions also do not happen on a smart contract platform (as smart contracts are not a first-class citizen of Substrate-based chains), so there is no concept of gas to meter transactions with.
+
+To seemingly complicate matters further, there are parachain transactions (within the parachain, as well as parachain-to-relay-chain), parathread transactions, and smart contract transactions in a smart contract pallet like the EVM pallet or the Ink pallet. This document will explain how it all fits together and how we meter transactions in Polkadot.
```

Comment:
Where do we class XCMP messages?

Reply:
I don't think those count here because other chains don't have something similar. This document is meant to only compare transactions to their implementations in other chains that people might be familiar with. That said, if XCMP messages change something here drastically, let's include them. Do we know *anything* about their fees / inclusions / types / frequency, anything concrete we could state here?

Reply:
So it turns out XCMP channels will "stay open" and require a DOT deposit of some kind (Gav's words). Since this is still all up in the air, I'd rather avoid the topic.